### ISSUE TITLE
The predicted coordinates remain unchanged

### ISSUE THREAD

**Issue by Baxkiller** [cooelf/Auto-GUI#18]
Hi, thanks for your awesome work!

I have tried to reproduce your result and found the predicted tap coordinate of test dataset always been the same(as shown below).
![91ebeca48e40d892ff22647ab25f764](https://github.com/user-attachments/assets/1f61107b-8c9f-48c2-a014-c24a9ba381d0)

The hyperparameter settings at runtime are as follows:
```bash
python main.py --data_root blip \
    --model FLAN-ALPACA-BASE \
    --epoch 10 --lr 1e-5 \
    --user_msg seq_all_hist1.0_hist4_future4_cuda1_evalstep2_imghist_db \
    --img_type blip --img_dim 1408 \
    --bs 8 --eval_bs 16 --input_len 512 --output_len 256 --eval_acc 40 \
    --transform_axis --warmup_ratio 0.05 \
    --all_data 1.0 \
    --use_history 4 \
    --use_future 4 \
    --eval_subset AITZ_processed/blip/all_blip \
    --output_dir experiments \
    --device cuda:0 \
    --seed 3407 \
    --use_img_history
```

I tried different hyperparameters, but the above problem still persists, just in different coordinates.

Do you know why this happens? Thanks!

**Comment by cooelf** [comment]
Hi, I noticed that you passed the directory of ‘AITZ_processed’ dataset. Did you use the AITZ dataset instead of our processed AITW dataset?

**Comment by Baxkiller** [comment]
Yes! I processed the AITZ dataset with reference to utis_data.py and fetch_features.py to ensure that the fields and meanings of the processed dataset are consistent with the fields and meanings of the dataset you provided. The `data` field of the processed data dictionary contains the following keys, which are finally stored according to the dataset structure mentioned in the readme:

```python
keys = [
    "goal",
    "step_id",
    "image",
    "ui_positions",
    "ui_text",
    "ui_type",
    "result_touch_yx",
    "result_lift_yx",
    "result_action"
]
```

Could this problem be caused by the training data volume in the AITZ dataset being significantly smaller compared to the AITW training dataset?

**Comment by cooelf** [comment]
Not sure if there is a mismatch between the coordinate style. I am not familiar with AITZ. Could you check the the specific transformed yx coordinates for AITZ? Maybe there is some problems during the transformation for the gold yx, which makes the model learn to predict everything to [0.2, 0.5].

**Comment by Baxkiller** [comment]
You're right! I followed your advice and checked all the `target_touch` and `target_lift` coordinates in the AITWDatasetImg instances, and I found that none of them are [0.2, 0.5], and the `labels` do not contain the tokenized [0.2, 0.5] (i.e., [18189, 6, 3, 12100]). BUT when I checked the labels passed to the model's forward function, I found that [18189, 6, 3, 12100] appears with a very high frequency, which I think is the source of the problem.

Thank you for your reminder.
